### DISCUSSION TITLE
Hyperlinks

### DISCUSSION THREAD

**Discussion by pi43r** [uxbox/uxbox#1524] [Ideas]
First of all: Great application! 

Use Case: Before designing an application I use figma for a basic storyboard, collecting images and links to other websites. Converting a url to a hyperlink makes it easier to quickly open a page for reference.

Implementation: Figma uses the shortcut ctrl+k, which is already in use at penpot for making components. The other option is a link-button in the top bar. The url is parsed automatically.

![image](https://user-images.githubusercontent.com/3011806/151890890-be3fa76b-e56b-417d-8ff8-24e63ffc9846.png)

**Comment by hirunatan** [comment]
In the latest Penpot release, we added support for interaction actions in the prototyping tab. And one of the possible actions is to open an external URL: https://help.penpot.app/user-guide/prototyping/#prototyping-actions

This is not as automatic as in Figma, but I think it may suit your needs.

**Comment by pi43r** [comment]
That's a nice feature, but I don't think it's suitable in my case.
The links are only accessible from the prototype, which means I am way faster just copy pasting the url into the browser. 

Also I might have found a bug in firefox. Klicking on the url in the prototype opens a new tab without the address.

**Comment by hirunatan** [comment]
The bug with the new tab should be fixed with the yesterday's release.

For the hyperlinks, it may be useful, you are true. I've created an enhancement issue for this: https://tree.taiga.io/project/penpot/issue/2735

**Comment by ryanbreen** [comment]
Hopefully this doesn't expand too far beyond the bounds of the original request, but I would like to have hyperlinks within text elements themselves.  That is, if I have a paragraph of text, I may want to specify a selection that should be a hyperlink with its own interaction (rather than have the entire text area be the link).

**Reply by illright** [reply]
I would like to add to this that I want these hyperlinks to be available in the PDF export. I used Figma to design PDFs with links before, and I'd like to do the same with Penpot

**Comment by jasonbradberry** [comment]
This is a big one for me. We're using Penpot to create design documentation, and external links are a key part of this.

**Comment by orbiting-station** [comment]
This would be also very useful for pdf exports.

**Reply by GDivino** [reply]
For the specific use case of designing resumes/CVs and exporting to PDFs, being able to add hyperlinks to texts would be very useful. +1 on this feature.
